docs/transpile/passmanagers.ipynb

```diff
@@ -0,0 +1,221 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    " # Transpiler stages and preset pass managers\n",
+    "\n",
+    "Qiskit includes functions to build preset `PassManager` objects.  These are objects that manage a set of different `Passes`, which can include things like transforming the operations in your quantum circuit, optimizing the selection of qubits to be used on the target backend, and other optimizations that improve the overall efficiency of your circuit and suppress the effects of noise.  This page details the default passes when running `qiskit.compiler.transpile()` or when setting the `optimization_level` of a Runtime object.\n",
+    "\n",
+    "\n",
+    "When the `transpile()` function is called, a preset `StagedPassManager` object is created and then executed.  The four different preset pass managers range from very light to heavy optimization, which is specified by the `optimization_level` argument that can range from 0 to 3.  Higher optimization levels take more time and computational effort but may yield more optimal circuits.  Optimization level 0 is intended for device characterization experiments and, as such, only maps the input circuit to the constraints of the target backend, without performing any optimizations.  Optimization level 3 spends the most effort optimizing the circuit.  However, as many of the optimization techniques in the transpiler are heuristic-based, spending more computational effort does not always result in an improvement in the quality of the output circuit.  For specific details on the preset pass managers, look to the [API reference section](../api/qiskit/transpiler_preset).\n",
```

Comment:
> Higher optimization levels take more time and computational effort but may yield more optimal circuits.  Optimization level 0 is intended for device characterization experiments and, as such, only maps the input circuit to the constraints of the target backend, without performing any optimizations.  Optimization level 3 spends the most effort optimizing the circuit.  However, as many of the optimization techniques in the transpiler are heuristic-based, spending more computational effort does not always result in an improvement in the quality of the output circuit.

I think you can cut this long sentence and instead add a CTA to check out the transpiler defaults page (https://github.com/Qiskit/documentation/pull/267) which is going to talk about the optimization level in more depth

Reply:
Done here https://github.com/Qiskit/documentation/pull/283/commits/580ffb43b2d958026646a3f32e3e7e093fdd825b